evalml/problem_types/problem_types.py

```diff
@@ -11,11 +11,14 @@ class ProblemTypes(Enum):
     """Multiclass classification problem."""
     REGRESSION = 'regression'
     """Regression problem."""
+    TIME_SERIES_REGRESSION = 'time_series_regression'
```

Comment:
This is a nit-pick, but: in our docs examples we typically type out `problem_type='multiclass'` etc. Is it easier for people to type/remember `problem_type='time series regression'` or `problem_type='time_series_regression'`?

Reply:
Good point! Avoiding the underscores seems easier to type so I changed it to that hehe.